js/script.js

```diff
@@ -0,0 +1,81 @@
+const menu = document.querySelector('.second-header');
+function showMenu() {
+  menu.style.display = 'block';
+}
+
+function cancelMenu() {
+  menu.style.display = 'none';
+}
+
```

Comment:
- [ ] [OPTIONAL] While trying to check responsiveness I encountered a bug. If you click on the hamburger menu and expand the screen to the desktop version, the `navbar` links just move down from their original position. Also if you go back to the mobile version, and close the open hamburger menu and expand the screen back to the desktop version, the nav links disappear entirely. I would highly suggest you to fix this bug as these projects are gonna be used to display on your resume, portfolio, linkedIn etc and left out bugs would indicate that you are lacking attention to detail.

https://user-images.githubusercontent.com/44798044/233542272-f8a82f49-fad8-4d0e-ba3b-49df2f786408.mov